package.json

```diff
@@ -27,15 +27,20 @@
     "audit": "yarn npm audit --recursive --all --environment production",
     "add:client": "yarn workspace @labrute/client add",
     "add:server": "yarn workspace @labrute/server add",
+    "sfx:wsl:generate": "cd ./client/public/sfx && audiosprite $(find . -type f -name '*.mp3' -o -name '*.wav') -o sfx -u . && mv ./sfx.json ./../../src/assets/",
+    "sfx:generate": "cd ./client/public/sfx && audiosprite -o sfx -u . $(ls **/*.mp3 **/*.wav | grep -v 'background.mp3' | grep -v 'unused/' ) && mv ./sfx.json ./../../src/assets/",
     "remove:client": "yarn workspace @labrute/client remove",
     "remove:server": "yarn workspace @labrute/server remove",
     "run:retrieveFirstBonus": "cd server && yarn run:retrieveFirstBonus",
     "run:fixErroredBonuses": "cd server && yarn run:fixErroredBonuses"
   },
   "devDependencies": {
     "@eternaltwin/cli": "^0.14.3",
+    "@types/audiosprite": "^0",
     "@types/node": "^18.11.15",
+    "audiosprite": "^0.7.2",
     "concurrently": "^7.6.0",
+    "ffmpeg": "^0.0.4",
```

Comment:
Same here, ffmpeg isn't used since we're installing it outside the scope of the project